src/main/java/net/datafaker/providers/base/Medical.java

```diff
@@ -2,13 +2,20 @@
 
 /**
  * @since 0.8.0
+ * @deprecated since 2.3.0. This faker is deprecated due to migration
+ * to healthcare-specific aggregated into {@link net.datafaker.providers.healthcare.HealthcareFaker} fakers with similar methods.
  */
+@Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
If we are going to remove it
what is the replacement for `Medical#diseaseName` ?

Reply:
I will add it in next PR, cause it touches `Disease` which I mentioned.